eng/pipelines/templates/variables/globals.yml

```diff
@@ -1,7 +1,8 @@
 variables:
+  MAVEN_CACHE_FOLDER: $(Pipeline.Workspace)/.m2/repository
   DocWardenVersion: '0.7.1'
   WagonOptions: '-Dmaven.wagon.rto=300000 -Dmaven.wagon.http.retryHandler.count=3'
-  DefaultOptions: '--batch-mode --fail-at-end --settings eng/settings.xml $(WagonOptions)'
+  DefaultOptions: '-Dmaven.repo.local=$(MAVEN_CACHE_FOLDER) --batch-mode --fail-at-end --settings eng/settings.xml $(WagonOptions)'
```

Comment:
Was this done as the cache folder may be different? Is this to resolve the `sdk-build-tools` missing?

Reply:
We are passing this so that it is consistent across all our invocations and the caching task. If we don't pass it we might be making incorrect assumptions about which/where the m2 repository lives.